Userland/Libraries/LibGfx/ImageFormats/DICOMLoader.cpp

```diff
@@ -115,6 +162,42 @@ class DICOMLoadingContext {
         return {};
     }
 
+    ErrorOr<void> read_optional_metadata(DataElement const& element)
+    {
+        // Study Date
```

Comment:
The comments in this method (except for the spec link) seem a bit pointless

Reply:
I originally plan to add the spec link for every single element but the spec is such a maze that I gave up on looking for them.
FWIW, the comments are the exact name of the field in the spec. Maybe I should put them between quotes? I will delete them otherwise, I'll let you decide.

Reply:
Ah yeah, if it's from the spec then using quotes makes that clear.

Reply:
Done!